File: Zeze/Arch/LogoutEventArgument.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Zeze.Builtin.Online;

namespace Zeze.Arch
{
    public class LogoutEventArgument : EventArgs
    {
        public string Account { get; set; }
        public string ClientId { get; set; }
        public BOnline OnlineData { get; set; }
    }
}

File: ZezeJava/ZezeJava/src/main/java/Zeze/Arch/LogoutEventArgument.java
package Zeze.Arch;

import Zeze.Util.EventDispatcher;
import org.jetbrains.annotations.NotNull;

public class LogoutEventArgument implements EventDispatcher.EventArgument {
	public final @NotNull String account;
	public final @NotNull String clientId;

	public LogoutEventArgument(@NotNull String account, @NotNull String clientId) {
		this.account = account;
		this.clientId = clientId;
	}
}
